src/App.js

```diff
@@ -1,26 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import { BrowserRouter, Switch, Route} from 'react-router-dom';
```

Comment:
In general, we're destructing each property in a new line if the property count is more than 1, to increase readability. Like the following;

```suggestion
import {
  BrowserRouter,
  Switch,
  Route
} from 'react-router-dom';
```